parameterized-client/src/hook/view.js

```diff
@@ -0,0 +1,67 @@
+import { Provider, connect } from 'react-redux';
+import { createStore } from 'redux';
+import React from 'react';
+import { jobs, addJob } from "./state-reducers";
+import { Job } from "./job";
+
+window.parameterizedbuilds = window.parameterizedbuilds || {};
+
+let AddJob = ({ dispatch }) => {
+    return (
+        <a>
+            <span className="aui-icon aui-icon-small aui-iconfont-list-add" onClick={() => {
+                dispatch(addJob());
+            }} />
+        </a>
+    )
+};
+AddJob = connect()(AddJob);
+
+
+let JobList = ({jobs, errors}) => {
+    return <div>
+        {
+            jobs.map(job =>
+                <Job id={job.id} errors={errors}/>
+            )
+        }
+    </div>
+};
+
+const jobListStateInjector = (state, ownProps) => {
+    return {
+        jobs: state,
+        errors: ownProps.errors
+    }
+};
+JobList = connect(jobListStateInjector)(JobList);
+
+class App extends React.Component{
```

Comment:
Any reason to not just use an SFC here?

Reply:
Nope. I'm just not that familiar with react yet. If you have any other suggestions on improving this code whether it be stylist, code organization, or something more substantial I'd love some guidance.